### PR TITLE
 Solution of bugs

### PR DESCRIPTION
**Pull request by iAldrich23xX**
I solved some errors of the official plugin as well as the crashdump that happened with the ids and legacy_id

### CODE REVIEW

File: src/synapsepm/SynapsePM.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function onEnable() {
 
 //        $this->useLoadingScreen = (bool)$this->getConfig()->get('loadingScreen', true);
 
-        $this->getServer()->getCommandMap()->register("stransfer", new TransferCommand());
+        //$this->getServer()->getCommandMap()->register("stransfer", new TransferCommand());
```
Comment (PetteriM1) [inline src/synapsepm/SynapsePM.php]: Why?

File: src/synapsepm/utils/Utils.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 class Utils {
 
-    const NUKKIT_RUNTIMEID_TABLE = "https://raw.githubusercontent.com/NukkitX/Nukkit/master/src/main/resources/runtimeid_table.json";
+    const NUKKIT_RUNTIMEID_TABLE = "http://host.firebt.net/blocks.json";
```
Comment (PetteriM1) [inline src/synapsepm/utils/Utils.php]: Umm... Shouldn't we be using the original?

File: src/synapsepm/Player.php [hunk 3]
```diff
@@ -90,7 +94,7 @@ public function handleDataPacket(DataPacket $packet) {
 
     public function onUpdate(int $currentTick): bool {
         if ((microtime(true) - $this->lastPacketTime) >= 5 * 60) {
-            $this->close('', 'timeout');
+            $this->close('', 'Kicked by Server reason: AFK');
```
Comment (PetteriM1) [inline src/synapsepm/Player.php]: Timeout isn't same as AFK

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```
Comment (PetteriM1) [inline _config.yml]: What is that?

File: src/synapsepm/Synapse.php [hunk 5]
```diff
@@ -123,7 +124,7 @@ public function shutdown() {
         if ($this->verified) {
             $pk = new DisconnectPacket();
             $pk->type = DisconnectPacket::TYPE_GENERIC;
-            $pk->message = 'Server closed';
+            $pk->message = 'Server restarting!';
```
Comment (PetteriM1) [inline src/synapsepm/Synapse.php]: ... Just why?

File: src/synapsepm/utils/Utils.php [hunk 6]
```diff
@@ -32,17 +32,17 @@ public static function initBlockRuntimeIdMapping() {
                     ]
                 ]
             )), true);
-
+
```
Comment (PetteriM1) [inline src/synapsepm/utils/Utils.php]: Oof

File: src/synapsepm/Synapse.php [hunk 7]
```diff
@@ -4,6 +4,7 @@
 namespace synapsepm;
 
 use pocketmine\event\player\PlayerCreationEvent;
+use pocketmine\event\player\PlayerQuitEvent;
```
Comment (PetteriM1) [inline src/synapsepm/Synapse.php]: What is this for?

File: src/synapsepm/Player.php [hunk 8]
```diff
@@ -63,8 +66,9 @@ public function handleLoginPacket(PlayerLoginPacket $packet) {
     public function forceSendEmptyChunks() {
         foreach ($this->usedChunks as $index => $true) {
             Level::getXZ($index, $chunkX, $chunkZ);
-
+
```
Comment (PetteriM1) [inline src/synapsepm/Player.php]: Oof

File: src/synapsepm/Player.php [hunk 9]
```diff
@@ -4,6 +4,9 @@
 namespace synapsepm;
 
 use pocketmine\level\Level;
+use pocketmine\network\mcpe\protocol\AvailableActorIdentifiersPacket;
+use pocketmine\network\mcpe\protocol\DataPacket;
+use pocketmine\network\mcpe\protocol\ActorEventPacket;
```
Comment (PetteriM1) [inline src/synapsepm/Player.php:9]: Dublicate

**Review by PetteriM1 — CHANGES_REQUESTED** [review]

File: src/synapsepm/utils/Utils.php (hunk 2 above)
Comment (iAldrich23xX) [reply src/synapsepm/utils/Utils.php]: With the original it generates a bug in legacy_id for now you can use my website as you repair that failure of yours

**Pull request closed without merging**